src/winter_onboarding_2021/fleet_management_service/specs.clj

```diff
@@ -74,19 +77,31 @@
                       email-gen))
 (s/def :users/role #{"admin" "manager" "driver"})
 (s/def :users/password (s/and string? not-empty))
+(s/def :users/invite-id uuid?)
+(s/def :users/org-id uuid?)
 
 (s/def ::users
   (s/keys :req [:users/name
                 :users/role
                 :users/email
                 :users/password]))
 
+(s/def ::user-with-invite-token
+  (s/keys :req [:users/name
+                :users/role
+                :users/email
+                :users/password
+                :users/org-id
+                :users/invite-id]))
+
 (s/def ::users-all-attr
   (s/keys :req [:users/id
                 :users/name
                 :users/role
                 :users/email
-                :users/password]))
+                :users/password
+                :users/invite-id
+                :users/org-id]))
 
```

Comment:
The code in these specs are getting a little repetitive. Could use `clojure.spec.alpha/merge`? 

https://clojuredocs.org/clojure.spec.alpha/merge

Reply:
Tried using this, was breaking some changes, will look at this in a saperate PR later.